### PR TITLE
Feature/profile

### PR DESCRIPTION
**Pull request by AnusreeSajeevan**
Implemented view profile details, logout in profile screen.

### CODE REVIEW

**Review by KrupaDesai22 — APPROVED** [review]
looks good

File: user/account/src/main/java/in/projecteka/jataayu/user/account/ui/fragment/ProfileFragment.kt [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        viewModel.isEditMode.observe(this, Observer {
+//            if (it) {
+//                binding.spinnerListener = this
+//            } else {
+//                binding.spinnerListener = null
+//            }
+//            binding.spinnerYob.isEnabled = it
+
+//            binding.spinnerYob.getChildAt(0)?.let {
+//                (binding.spinnerYob.getChildAt(0) as TextView).setTextColor(
```
Comment (mahabaleshwarhnr) [inline user/account/src/main/java/in/projecteka/jataayu/user/account/ui/fragment/ProfileFragment.kt]: can you remove the comment?

File: user/account/src/main/java/in/projecteka/jataayu/user/account/viewmodel/CreateAccountViewModel.kt [hunk 2]
```diff
@@ -147,10 +147,10 @@ class CreateAccountViewModel(private val repository: UserAccountsRepository,
 
     private fun getGender(): String {
         return when (genderCheckId) {
-            R.id.gender_chip_male -> "M"
-            R.id.gender_chip_female -> "F"
-            R.id.gender_chip_other -> "O"
-            else -> "O"
+            R.id.gender_chip_male -> PreferenceRepository.GENDER_MALE
+            R.id.gender_chip_female -> PreferenceRepository.GENDER_FEMALE
```
Comment (mahabaleshwarhnr) [inline user/account/src/main/java/in/projecteka/jataayu/user/account/viewmodel/CreateAccountViewModel.kt:151]: It is good to move this to viewmodel. you can test this too

File: user/account/src/main/java/in/projecteka/jataayu/user/account/ui/fragment/ProfileFragment.kt (hunk 1 above)
Comment (AnusreeSajeevan) [reply user/account/src/main/java/in/projecteka/jataayu/user/account/ui/fragment/ProfileFragment.kt]: Removed in the latest commit.

**Pull request merged**